src/components/progress.js

```diff
@@ -75,13 +76,15 @@ class MiniMediaPlayerProgress extends LitElement {
             </div>
           </div>
         ` : ''}
-        <paper-progress class=${this.classes}
-          value=${this.seekProgress || this.progress}
-          max=${this.duration}>
-        </paper-progress>
+        <div class='progress-bar' style=${this.progressBarStyle()}></div>
       </div>
     `;
   }
+   progressBarStyle() {
+    return styleMap({
+      width: `${(this.seekProgress || this.progress / this.duration) * 100}%`
```

Comment:
```suggestion
      width: `${((this.seekProgress || this.progress) / this.duration) * 100}%`
```
The parentheses are missing, so `seekProgress` is not calculated correctly.